code/src/index.css

```diff
@@ -1,13 +1,284 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
```

Comment:
Did you two have any problem with formatting incompatibilities due to Prettier? Just curious (we had some issues when Prettier converted double to single quotes).